system/Model.php

```diff
@@ -131,7 +131,7 @@ class Model extends BaseModel
     /**
      * Primary Key value when inserting and useAutoIncrement is false.
      *
-     * @var int|string|null
+     * @var mixed|null
```

Comment:
regardless with type change, the Rector change `null|mixed` is changed back to `mixed|null` by php-cs-fixer

Reply:
I skipped TypedPropertyFromAssignsRector in various location due to flip-flop union mixed position https://github.com/codeigniter4/CodeIgniter4/pull/6942/commits/21052eac4ebcd88206613fc59603968a1f5aa37b) https://github.com/codeigniter4/CodeIgniter4/pull/6942/commits/21052eac4ebcd88206613fc59603968a1f5aa37b

Reply:
I create issue https://github.com/rectorphp/rector/issues/7640 for it.

Reply:
I create PR:

- https://github.com/rectorphp/rector-src/pull/3160

to patch it. I will revert manually in other areas for now:

```
            __DIR__ . '/tests/system/API/ResponseTraitTest.php',
            __DIR__ . '/tests/system/Honeypot/HoneypotTest.php',
            __DIR__ . '/tests/system/Filters/HoneypotTest.php',
```